### ISSUE TITLE
Proposed Analysis: Molecular subtyping for LGAT 

### ISSUE THREAD

**Issue by jaclyn-taroni** [AlexsLemonade/OpenPBTA-analysis#631]
#### What are the scientific goals of the analysis?

Breaking things out from https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/627#issuecomment-598789232 - **Molecular subtyping of LGAT** into LGG BRAF fusion, LGG BRAF V600E, or LGG BRAF WT

#### Analysis steps

1. Subset the consensus mutation file mentioned below to biospecimens where `short_histology == LGAT` and commit them to the repo. This may be necessary for any subsequent steps to pass CI, otherwise the number of samples that meet this criterion may be low (less of a concern with LGAT, though). This subsetting step will not go into CI. Check out [`molecular-subtyping-ATRT`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/7e000418b106e34040325d2fbd642f63080966c8/analyses/molecular-subtyping-ATRT) for an example. We'll want to set things up such that if someone runs things locally, the subset files will be generated using the version of the data symlinked in `data/`.

2. Identify LGAT biospecimens with _BRAF_ fusions using `pbta-fusion-recurrently-fused-genes-bysample.tsv`. _KIAA1549--BRAF_ fusions are of particular interest, but to get specific fusions we would have to use `pbta-fusion-putative-oncogenic.tsv`.

3. Identify LGAT biospecimens with BRAF V600E mutations in `pbta-snv-consensus-mutation.maf.tsv.gz`.

4. Create a final table that contains:

| `Kids_First_Participant_ID` | `sample_id` | `Kids_First_Biospecimen_ID_DNA` | `Kids_First_Biospecimen_ID_RNA` | `molecular_subtype` |
|----------------------------|-------------|------------------------------------|-----------------------|---------------|
| PT_XXXXXX | 7316-XXXX | BS_XXXXXXXX | BS_XXXXXXXX | LGG, BRAF wildtype | 
| PT_XXXXXX | 7316-XXXX | BS_XXXXXXXX | BS_XXXXXXXX | LGG, BRAF V600E |
| PT_XXXXXX | 7316-XXXX | BS_XXXXXXXX | BS_XXXXXXXX | LGG, BRAF fusion | 
| PT_XXXXXX | 7316-XXXX | BS_XXXXXXXX | BS_XXXXXXXX | LGG, BRAF V600E/fusion |   

Quoting [Penman et al.](https://dx.doi.org/10.3389%2Ffonc.2015.00054):

> The BRAF V600E mutant and the KIAA1549:BRAF fusion are generally mutually exclusive with only a few cases reported with both the fusion and the V600E mutation

But the logic for constructing the final table account for the fact that it is possible for both to be present.

#### What input data are required for this analysis?

```
pbta-histologies.tsv
pbta-snv-consensus-mutation.maf.tsv.gz
pbta-fusion-recurrently-fused-genes-bysample.tsv
```

#### Who will complete the analysis (please add a GitHub handle here if relevant)?

@kgaonkar6 was mentioned on the linked comment – let me know if this is okay!

#### What relevant scientific literature relates to this analysis?

This is beyond my area of expertise but the following seem like they may be relevant starting points:

* [Penman et al. _Front Oncol._ 2015.](https://dx.doi.org/10.3389%2Ffonc.2015.00054)
* [Jones et al. _Neuro Oncol._ 2018.](https://doi.org/10.1093/neuonc/nox141)

**Comment by kgaonkar6** [comment]
ok @jaclyn-taroni ! Thanks for providing the details above I'll start working on this.

**Comment by jaclyn-taroni** [comment]
Closed via #634 and #642 - anything left with LGAT is subsumed by #645.
